draft-ietf-httpbis-digest-headers.md

```diff
@@ -999,6 +1036,20 @@ the transport layer that protects HTTP fields.
 A `Digest` field using NOT RECOMMENDED digest-algorithms SHOULD NOT be used in
 signatures.
 
+## Usage in trailers
+
+When used in trailers, the receiver gets the digest value after the payload body
+and may thus be tempted to process the data before validating the digest value.
+Instead, data should only be processed after validating the Digest.
+
+If received in trailers, Digest MUST NOT be discarded;
```

Comment:
@LPardue sorry Lucas, could you please check this part I stumbled upon re-reading http-messaging-latest?